### ISSUE TITLE
Make CPTs' custom meta storage option compatible with maps

### ISSUE THREAD

**Issue by killian-santos** [CrocoBlock/suggestions#7687]
Hello,

I recently tested the "Custom Meta Storage" option of CPTs with map elements.

However, when the coordinates are stored in the CPT table, the geo search query is unable to retrieve them, making it impossible to display them on a map.

Please make the custom meta storage option compatible with the use of maps.

Thank you.

**Comment by ihslimn** [comment]
Could you please provide more info?
I created a post type with custom storage enabled, added a map meta field to it, and it works with Map Listing just fine.
Please, describe step by step the process of recreating the problem.

Also, have you contacted our support team with this issue?

**Comment by killian-santos** [comment]
Hi @ihslimn,

I contacted your support team 2 weeks ago. Detailing the following steps:

1. I have created a CPT with latitude + longitude fields: https://www.screenpresso.com/=Imc6g
2. I created a smart filter “Location & distance”.
3. Then on a page, I integrated the smart filter with a map: https://www.screenpresso.com/=e00rc

However, following these steps, I couldn't see all the points on the map.

Here is the answer I received: 

> "Hi there, got the response from the tech team
> It seems that the issue caused after enabling this option ​https://i.imgur.com/mfoiSNa.png
> only meta of a new posts will be transferred to a new table https://i.imgur.com/boDb9mT.png
> f we create the metabox and add the coordinates there, then it works fine"

**Comment by ihslimn** [comment]
Ah, so the issue is not with the custom storage.
The issue is that you should resave posts manually for meta fields to actually get stored into a custom table, this - for now - is the only way to get old post values to be saved into it.

So the custom storage is compatible with maps, it is just that you had post type with custom storage disabled, and after enabling it no values are transfered - which is **the expected behavior**, and it is even mentioned in the description of the option

**Comment by killian-santos** [comment]
Yes, except that I also did it and saw the values in the table. But even so, the map didn't show everything.

**Comment by killian-santos** [comment]
Hi @ihslimn, I have opened a new support ticket with a detailed case: #191996. Can you take a look at this?

**Comment by aalexandr21** [comment]
The issue with the Custom Meta Storage and the Location & Distance filter was fixed in JetEngine 3.6.1
